### ISSUE TITLE
2FA On docker?

### ISSUE THREAD

**Issue by walkxcode** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#386]
Hey, I'm running this on docker. But it's asking me for my 2FA, how do I put this in???

**Comment by Tkd-Alex** [comment]
Try to start the container with the `-it` flag

**Comment by walkxcode** [comment]
I want to run the container in the background, you can't do that with an interactive shell

**Comment by Tkd-Alex** [comment]
Yes but you should run only the first time with `-it` flag, login, create the cookie and then restart without the flag
